client/src/shared/services/ajaxFactory.js

```diff
@@ -17,9 +17,9 @@
 
     ajaxObj.isOrderComplete = function(transactionId) {
       return $http({
-        method: 'GET',
-        url: serverUrl + '/order/details',
-        params: { id: transactionId }
+        method: 'POST',
+        url: serverUrl + '/hero/order/details',
+        data: {'transactionId': transactionId}

```

Comment:
For curly braces there should be an space after open brace and before closing brace.
`{ 'transactionId': transanctionId }`
Same for double braces
`{{ ...stuff.... }}`